river/command/focused_view_opacity.zig

```diff
@@ -0,0 +1,56 @@
+// This file is part of river, a dynamic tiling wayland compositor.
+//
+// Copyright 2020 Isaac Freund
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+const std = @import("std");
+
+const Config = @import("../Config.zig");
+const Error = @import("../command.zig").Error;
+const Output = @import("../Output.zig");
+const Seat = @import("../Seat.zig");
+
+pub fn opacity(
+    allocator: *std.mem.Allocator,
+    seat: *Seat,
+    args: []const []const u8,
+    out: *?[]const u8,
+) Error!void {
+    if (args.len < 2) return Error.NotEnoughArguments;
+    if (args.len > 2) return Error.TooManyArguments;
+
+    const op = std.fmt.parseFloat(f32, args[1]);
```

Comment:
You probably should use `try`.

Reply:
Also I think river uses `f64` everywhere, but I am not sure...

Reply:
wlroots exposes alpha as f32. So I carried that through to everywhere.

Reply:
I see. In that case this makes sense.